src/driver/gen_rpc_driver_tcp.erl

```diff
@@ -82,6 +83,29 @@ send(Socket, Data) when is_port(Socket), is_binary(Data) ->
             ok
     end.
 
+-spec send_async(port(), iodata()) -> ok | {error, term()}.
+send_async(Socket, Data) when is_port(Socket) ->
+    case send_tcp_data(Socket, Data) of
+        {error, Reason} ->
+            ?log(error, "event=send_async_failed socket=\"~s\" reason=\"~p\"", [gen_rpc_helper:socket_to_string(Socket), Reason]),
+            {error, {badtcp,Reason}};
+        ok ->
+            ?log(debug, "event=send_async_succeeded socket=\"~s\"", [gen_rpc_helper:socket_to_string(Socket)]),
+            ok
+    end.
+
+-if(?OTP_RELEASE >= 26).
+send_tcp_data(Sock, Data) ->
+    gen_tcp:send(Sock, Data).
+-else.
+send_tcp_data(Sock, Data) ->
+    try erlang:port_command(Sock, Data) of
```

Comment:
Q: prior to OTP 26, we avoided `gen_tcp:send` because it lacked the "reference trick"?
For OTP 26+, `send_async` seems to be same as `send`?

Reply:
yes, the new code has a monitor reference sent to the port.
https://github.com/erlang/otp/commit/f17f8024a41e8b6fd2a8cb5be765d6e714e3e8f9